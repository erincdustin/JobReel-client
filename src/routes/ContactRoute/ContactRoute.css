.content-wrapper {
  display: flexbox;
  justify-content: space-evenly;
  align-content: center;
}

.person-card {
  display: inline-block;
  border: 2px solid #eeccee;
  width: 200px;
  height: 150px;
  margin: 1em;
  padding: 0.5em;
  border-radius: 10px;
}

.person-card:hover {
  background: #3C824E;
  transition: 0.3s ease-in;
}

.person-contact-info {
  width: 40%;
  margin-left: 60%;
}