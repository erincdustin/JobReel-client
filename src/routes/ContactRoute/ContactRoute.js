import React from 'react';
import './ContactRoute.css';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
const propTypes = {};

const defaultProps = {};

export default class ContactRoute extends React.Component {
  render() {
    return (
        <div className='content-wrapper'>
          <div className='person-card'>
            <div className='person-name'>
              <h3>Aldiyar Batyrbekov</h3>
            </div>
            <div className='role'>
              <h4>Design Lead</h4>
            </div>
            <div className='person-contact-info'>
              <a href='https://www.linkedin.com/in/aldibatyr/'><FontAwesomeIcon size='2x' icon={['fab', 'linkedin']}/></a>
              {' '}
              <a href='https://github.com/aldibatyr'><FontAwesomeIcon size='2x' icon={['fab', 'github-square']}/></a>
            </div>
          </div>
          <div className='person-card'>
            <div className='person-name'>
              <h3>Chris Fiander</h3>
            </div>
            <div className='role'>
              <h4>Product Manager</h4>
            </div>
            <div className='person-contact-info'>
              <a href='-empty'><FontAwesomeIcon size='2x' icon={['fab', 'linkedin']}/></a>
              {' '}
              <a href='-empty'><FontAwesomeIcon size='2x' icon={['fab', 'github-square']}/></a>
            </div>
          </div>
          <div className='person-card'>
            <div className='person-name'>
              <h3>Erin Dustin</h3>
            </div>
            <div className='role'>
              <h4>Project Manager</h4>
            </div>
            <div className='person-contact-info'>
              <a href='https://www.linkedin.com/in/erin-dustin/'><FontAwesomeIcon size='2x' icon={['fab', 'linkedin']}/></a>
              {' '}
              <a href='https://github.com/erincdustin'><FontAwesomeIcon size='2x' icon={['fab', 'github-square']}/></a>
            </div>
          </div>
          <div className='person-card'>
            <div className='person-name'>
              <h3>Jonathan Moore</h3>
            </div>
            <div className='role'>
              <h4>QA Lead</h4>
            </div>
            <div className='person-contact-info'>
              <a href='https://www.linkedin.com/in/jonathan-moore-73b24721/'><FontAwesomeIcon size='2x' icon={['fab', 'linkedin']}/></a>
              {' '}
              <a href='https://github.com/JonathanEdMoore'><FontAwesomeIcon size='2x' icon={['fab', 'github-square']}/></a>
            </div>
          </div>
          <div className='person-card'>
            <div className='person-name'>
              <h3>Michael Romero</h3>
            </div>
            <div className='role'>
              <h4>Product Manager</h4>
            </div>
            <div className='person-contact-info'>
              <a href='https://www.linkedin.com/in/michael-romero-854037147/'><FontAwesomeIcon size='2x' icon={['fab', 'linkedin']}/></a>
              {' '}
              <a href='https://github.com/michaelromero09'><FontAwesomeIcon size='2x' icon={['fab', 'github-square']}/></a>
            </div>
          </div>
        </div>
    );
  }
}

 ContactRoute.propTypes = propTypes;
 ContactRoute.defaultProps = defaultProps;