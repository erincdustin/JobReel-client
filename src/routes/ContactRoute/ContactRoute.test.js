import React from 'react'
import ReactDOM from 'react-dom'
import {BrowserRouter} from 'react-router-dom'
import ContactRoute from './ContactRoute'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('NotFoundRoute smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><ContactRoute /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('NotFoundRoute snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<ContactRoute />)
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})