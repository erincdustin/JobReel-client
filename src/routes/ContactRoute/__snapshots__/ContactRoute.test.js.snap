// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundRoute snapshot test renders the UI as expected 1`] = `
<div
  className="content-wrapper"
>
  <div
    className="person-card"
  >
    <div
      className="person-name"
    >
      <h3>
        Aldiyar Batyrbekov
      </h3>
    </div>
    <div
      className="role"
    >
      <h4>
        Design Lead
      </h4>
    </div>
    <div
      className="person-contact-info"
    >
      <a
        href="https://www.linkedin.com/in/aldibatyr/"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "linkedin",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
       
      <a
        href="https://github.com/aldibatyr"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "github-square",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </div>
  </div>
  <div
    className="person-card"
  >
    <div
      className="person-name"
    >
      <h3>
        Chris Fiander
      </h3>
    </div>
    <div
      className="role"
    >
      <h4>
        Product Manager
      </h4>
    </div>
    <div
      className="person-contact-info"
    >
      <a
        href="-empty"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "linkedin",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
       
      <a
        href="-empty"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "github-square",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </div>
  </div>
  <div
    className="person-card"
  >
    <div
      className="person-name"
    >
      <h3>
        Erin Dustin
      </h3>
    </div>
    <div
      className="role"
    >
      <h4>
        Project Manager
      </h4>
    </div>
    <div
      className="person-contact-info"
    >
      <a
        href="https://www.linkedin.com/in/erin-dustin/"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "linkedin",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
       
      <a
        href="https://github.com/erincdustin"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "github-square",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </div>
  </div>
  <div
    className="person-card"
  >
    <div
      className="person-name"
    >
      <h3>
        Jonathan Moore
      </h3>
    </div>
    <div
      className="role"
    >
      <h4>
        QA Lead
      </h4>
    </div>
    <div
      className="person-contact-info"
    >
      <a
        href="https://www.linkedin.com/in/jonathan-moore-73b24721/"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "linkedin",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
       
      <a
        href="https://github.com/JonathanEdMoore"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "github-square",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </div>
  </div>
  <div
    className="person-card"
  >
    <div
      className="person-name"
    >
      <h3>
        Michael Romero
      </h3>
    </div>
    <div
      className="role"
    >
      <h4>
        Product Manager
      </h4>
    </div>
    <div
      className="person-contact-info"
    >
      <a
        href="https://www.linkedin.com/in/michael-romero-854037147/"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "linkedin",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
       
      <a
        href="https://github.com/michaelromero09"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "github-square",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="2x"
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </div>
  </div>
</div>
`;
